### ISSUE TITLE
Sign message popup doesn't appear

### ISSUE THREAD

**Issue by joeelmahallawy** [ChainSafe/dappeteer#105]
**Describe the bug**
When clicking on a button that triggers a metamask popup to sign a message, nothing happens.

**To Reproduce**
// This should trigger a sign message popup on metamask
await page.click(
    ".Blockreact__Block-sc-1xf18x6-0:nth-child(1) > .Blockreact__Block-sc-1xf18x6-0 > .Blockreact__Block-sc-1xf18x6-0 > .UnstyledButtonreact__UnstyledButton-sc-ty1bh0-0 > .Iconreact__Icon-sc-1gugx8q-0"
  );
  await metamask.page.bringToFront(); 
  await metamask.approve(); // doesn't work
  await metamask.confirmTransaction(); // doesn't work
  await metamask.sign(); // doesn't work

**Expected behavior**
There should be a popup or at least metamask should be in sign state.

**Comment by Lawlzer** [comment]
Can confirm, this is still happening.

An easier way to get the error is to open your console and run this:
```
if (window.ethereum) {
  try {
    const accounts = await window.ethereum.request({ method: 'eth_requestAccounts' });
  } catch (error) {
    console.log(error); 
  }
}
```
Error code 4001 (User rejected the request)

**Comment by alaaan** [comment]
Anyone get any resolution with this?

**Comment by lutsykvad** [comment]
@alaaan try this one (works for me)
 try {
      await page.waitForSelector(this.connectToWalletButton, { visible: true });
      await page.click(this.connectToWalletButton);
      await metamask.approve();
    } catch {
      console.error("Metamask selector timeout");
    }

**Comment by BeroBurny** [comment]
should be fixed in 3.0.0-rc.0
